src/SourceBuild/content/eng/pipelines/sb-pr.yml

```diff
@@ -5,9 +5,8 @@ trigger: none
 pr:
   branches:
     include:
```

Comment:
Apparently this is also used for CI in 8.0.1xx and 9.0.1xxx

Reply:
Also, this isn't used except in the public project, so no need for internal/* branches.